### ISSUE TITLE
Remove console.info from codebase

### ISSUE THREAD

**Issue by MilippFeier** [j4ts/j4ts#63]
Hi,
i am not sure which of our dependencies is including j4ts, however, it is constantly printing the following to the browser console: 
```
Object { modCount: 0, array: (3) […] }
array: Array(3) [ "a", "b", "c" ]
modCount: 0
```
The console.info is in j4ts.js:2147:21.

```
(function (test) {
    class Test {
        static main(args) {
            console.info(java.util.Arrays.asList("a", "b", "c"));   // <<<< here
        }
        static assertEquals(o1, o2) {
            if (!(o1 === o2)) {
                throw new Error("invalid assertion: " + o1 + "!=" + o2);
            }
        }
```

**Comment by lgrignon** [comment]
Obviously this is a test class and should not be included in final bundle. Thanks for reporting @MilippFeier

**Comment by lgrignon** [comment]
I think this is already fixed in 2.0.0 @MilippFeier could you please give it a try? (It may require JSweet 3 though)

**Comment by MilippFeier** [comment]
Ahh, okay, Didnt check the version that is running. Nevermind.
